pkg/minishift/minishiftdata/minishiftdata.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package minishiftdata
+
+import (
+	"encoding/json"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+var (
+	minishiftDataPath string
+)
+
+type MinishiftData struct {
+	OcPath string `json:"OcPath"`
+}
+
+// Create minishift-data.json with empty data or return with data if exists
+func Create(machinesDirPath string) (*MinishiftData, error) {
```

Comment:
I think the naming is confusing;
  * Create an object?
  * Create the file? 

The function name and the package path itself do not do a good job at conveyong the intention if the functionality.

`minishiftdata` sounds like all the datafiles that are handled by Minishift... or is it metadata? WDYT?